### PR TITLE
Fix token prices cache

### LINKED ISSUE #2080: [carbon dashboard] The asset prices on data.klimadao.finance are not up to date
**Describe the bug**
On this page: https://data.klimadao.finance/fr/overview/price-of-digital-carbon
The prices are not up to date.

**investigation**

A direct call to the BE prices endpoint gives up to date values: `https://data.klimadao.finance/api/prices`
So there must be an issue with the page/request cache  configuration

### PR DESCRIPTION
**Pull request by biwano**
## Description

Fixes token prices cache. Cache duration is reduced to 1 minute.

This PR is part of a combo to improve accuracy of prices in Carbonmark and the carbon dashboard. This is the Carbon dashboard part.
This PR should be merged synchronously with this one: https://github.com/Carbonmark/monorepo/pull/82

## Related Ticket

Closes #2080

## Notes For QA

Specific pages, components or journeys that might be affected:
- Price of digital carbon

Relevant preview URLs:
- Carbon dashboard: https://carbon-git-carbonfixpricecache-klimadao.vercel.app
- Carbonmark: https://carbonmark-git-biwano-fix81-carbonmark.vercel.app/fr/retire

Other notes:
-

### CODE REVIEW

File: carbon/lib/constants.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ const api_url =
     : // Client side we use the proxy
       "/api";
 
+export const CACHE_DURATION = 3600;
+export const TOKENS_CACHE_DURATION = 60;
```
Comment (sprrwhwk) [inline carbon/lib/constants.ts:46]:
```suggestion
export const CACHE_DURATION_SECONDS = 3600;
export const TOKENS_CACHE_DURATION_SECONDS = 60;
```

Please add units here

**Review by sprrwhwk — DISMISSED** [review]
I still don't seem to get up-to-date prices on this branch

grafik

**Review by jabby09 — COMMENTED** [review]
@biwano 
It looks like #2049 is merged to staging so I'm comparing https://carbon-1y3hmiz4h-klimadao.vercel.app/fr/overview/price-of-digital-carbon to https://staging.carbonmark.com/projects?layout=grid&sort=price-lowest

Marketplace BCT pricing...
Screenshot 2024-01-09 at 9 07 15 AM

Klima data BCT pricing...
Screenshot 2024-01-09 at 9 08 01 AM

**Comment by biwano** [comment]
> @biwano It looks like #2049 is merged to staging so I'm comparing https://carbon-1y3hmiz4h-klimadao.vercel.app/fr/overview/price-of-digital-carbon to https://staging.carbonmark.com/projects?layout=grid&sort=price-lowest
> 
> Marketplace BCT pricing... Screenshot 2024-01-09 at 9 07 15 AM
> 
> Klima data BCT pricing... Screenshot 2024-01-09 at 9 08 01 AM

There is another caching issue on the data-pipelines side. I am waiting for the new code to be reviewed and deployed.

File: carbon/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "private": true,
   "sideEffects": false,
   "scripts": {
-    "dev": "next dev -p 3003",
+    "dev": "next dev -p 3004",
```
Comment (0xemc) [inline carbon/package.json:6]: ah.. is this meant to be here?

Comment (biwano) [reply carbon/package.json:6]: Hehe! No...

**Review by jabby09 — APPROVED** [review]

**Pull request merged**